pkg/addressbook/addressbook.go

```diff
@@ -90,15 +85,15 @@ func (s *store) Overlays() (overlays []swarm.Address, err error) {
 	return overlays, nil
 }
 
-func (s *store) Multiaddresses() (multis []ma.Multiaddr, err error) {
+func (s *store) Addresses() (multis []bzz.Address, err error) {
```

Comment:
multis should be renamed as it is no longer just multiaddresses

Reply:
This comment is left.